bin/README.md

```diff
@@ -113,14 +113,14 @@ This script is used to allow you to pull a changelog of a specific image between
 
 ## Usage
 To pull the changelog between two dates run the script like this:</br>
-```python3 trigger_changelog.py -u <github_username> -t <github_token> -ir <image_repository> -ib <image_branch (default: noetic-devel)> -sd <start_date> -ed <end_date>```</br>
```

Comment:
What did 'i' do in the first place?

Reply:
Extracted from `python3 --help`:

> -i     : inspect interactively after running script; forces a prompt even
         if stdin does not appear to be a terminal; also PYTHONINSPECT=x

Reply:
i had -ib and -ir to stand for image branch and image repo, but it didn't make sense